nautobot/dcim/api/serializers.py

```diff
@@ -743,23 +471,25 @@ def validate(self, data):
 
         return data
 
-    @extend_schema_field(NestedDeviceSerializer)
-    def get_parent_device(self, obj):
+    @extend_schema_field(DeviceBaySerializer)
+    def get_parent_bay(self, obj):
```

Comment:
Need to document this as an API change.

Reply:
Added as an entry in `v2-api-renamed-fields.yaml`